src/Microsoft.PowerShell.ConsoleHost/host/msh/Executor.cs

```diff
@@ -320,11 +322,346 @@ internal Collection<PSObject> ExecuteCommand(string command, out Exception excep
         {
             Dbg.Assert(!String.IsNullOrEmpty(command), "command should have a value");
 
+            // Experimental:
+            // Check for implicit remoting commands that can be batched, and execute as batched if able.
+            if (ExperimentalFeature.EnabledExperimentalFeatureNames.TryGetValue("PSImplicitRemotingBatching", out string actualValue))
```

Comment:
Please use `ExperimentalFeature.IsEnabled("PSImplicitRemotingBatching")`

Reply:
Will do.

Reply:
Should we collect all constants like "PSImplicitRemotingBatching" for all code base in one place?